src/SearchResults/index.js

```diff
@@ -510,7 +508,7 @@ function SearchResults(state, results) {
     });
   });
 
-  this.hierarchicalFacets = map(this.hierarchicalFacets, generateHierarchicalTree(state));
+  this.hierarchicalFacets = this.hierarchicalFacets.map(generateHierarchicalTree(state));
```

Comment:
object?

Reply:
> arg, I forgot that _.map and _.reduce also work on objects; should check if that's the case here...

Object.entries() works well for those cases

Reply:
Yep, will have to use object.keys in those (entries is edge, end helper supports ie9)

Reply:
This should be discussed with everyone about the browser support of helper, IE9 is really old. We can rely on polyfills. Like InstantSearch does

Reply:
I’d rather not add polyfills in a PR aimed at making the size smaller, but we can go further in removing methods if ie11 or edge is picked as target

Reply:
For the record: `this.hierarchicalFacets` is an array